CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+The following is a set of guidelines for contributing to ERT.
+
+## Ground Rules
+
+1. We use Black code formatting
+1. All code must be testable and unit tested.
+
+## Pull Request Process
+
+1. Work on your own fork of the main repo
+1. Push your commits and make a draft pull request using the pull request template.
+1. Check that your pull request passes all tests.
+1. When all tests have passed and your are happy with your changes, change your pull request to "ready for review"
```

Comment:
Do you refer to the label or the zenhub column? Because the later is not available for external people... Should we consider actively using the `Draft`?

Reply:
I'm not sure why zenhub was introduced into this PR, but it does not apply.

We use draft PRs all the time—we mark them as "Ready for review" all the time… No zenhub involved!